common/utils/convert-image-uri.js

```diff
@@ -45,94 +41,147 @@ function determineSrc(url: string): string {
 }
 
 function determineIfGif(originalUriPath) {
-  return originalUriPath.slice(-4) === '.gif';
+  return originalUriPath.includes('.gif');
 }
 
 function determineFinalFormat(originalUriPath) {
-  if (originalUriPath.slice(-4) === '.png') {
+  if (originalUriPath.includes('.png')) {
     return 'png';
   } else {
     return 'jpg';
   }
 }
 
-function convertPathToWordpressUri(originalUriPath, size) {
-  return originalUriPath + `?w=${size}`;
+type PrismicUriOpts = {|
+  auto?: string,
+  rect?: string,
+  w?: number | 'full',
+  h?: number,
+|};
+
+function prismicImageTemplate(baseUrl: string) {
+  const templateString = `${baseUrl}?auto={auto}&rect={rect}&w={w}&h={h}`;
+  const template = urlTemplate.parse(templateString);
+  return (opts: PrismicUriOpts) => {
+    return template.expand(opts);
+  };
 }
 
-function convertPathToIiifUri(originalUriPath, iiifRoot, size) {
-  const isFullSize = size === 'full';
-  const format = determineFinalFormat(originalUriPath);
-  return `${iiifRoot}${originalUriPath}/full/${size}${
-    isFullSize ? '' : ','
-  }/0/default.${format}`;
+type WordpressUriOpts = {|
+  width?: number | 'full',
```

Comment:
It doesn't look as though width is optional though?

Reply:
it would return an unsized image